tests/spec/core/webidl-spec.js

```diff
@@ -338,33 +344,46 @@ describe("Core - WebIDL", () => {
     );
   });
 
-  it("should handle constructors", () => {
-    let target = doc.getElementById("ctor-basic");
-    // Remove the header, as we are not interested in it.
+  it("handles constructors", async () => {
+    const body = `
+      <pre id='ctor-basic' class='idl'>
+        [Exposed=Window]
+        interface SuperStar {
+          constructor();
+          constructor(boolean bar, sequence&lt;double> foo);
+        };
+      </pre>
+    `;
+    const ops = makeStandardOps(null, body);
+    const doc = await makeRSDoc(ops);
+
+    const text = `
+[Exposed=Window]
+interface SuperStar {
+  constructor();
+  constructor(boolean bar, sequence<double> foo);
+};
```

Comment:
I wonder we could dedupe this, currently `body` and `text` are quite duplicated.

Reply:
It's a bit messy, because they are testing whitespace.